docs/components/concepts/process-instance-creation.md

```diff
@@ -27,28 +27,23 @@ This command creates a new process instance and immediately responds with the pr
 ![create-process](assets/create-process.png)
 
 <details>
-  <summary>Code example</summary>
-  <p>
-Create a process instance:
+   <summary>Create a process instance via Camunda 8 REST API</summary>
+   <p>
 
 ```
-zbctl create instance "order-process"
+curl -L 'http://localhost:8080/v2/process-instances' \
+-H 'Content-Type: application/json' \
+-H 'Accept: application/json' \
+-d '{
+  "processDefinitionKey": "2251799813685249”,
+  "processDefinitionVersion": 1
+}'
 ```
 
-Response:
```

Comment:
❓ Is there a reason not to show the response for this request here like it was shown before?

Reply:
I did not yet have a chance to run the API. If it's the same, I can add it back.

Reply:
Mapped the terms, adding in a new commit.